packages/grpc-protobufjs/src/index.ts

```diff
@@ -37,6 +37,8 @@ export interface MethodDefinition<RequestType, ResponseType> {
   requestDeserialize: Deserialize<RequestType>;
   responseDeserialize: Deserialize<ResponseType>;
   originalName?: string;
+  requestType: RequestType;
+  responseType: ResponseType;
```

Comment:
These type declarations aren't really semantically correct. `RequestType` and `ResponseType` are the types of objects that are passed to and received from the gRPC methods. In the case of this library, those are plain objects with a structure matching the declared proto message types, which is why the `ServiceDefinition` just declares them with `object, object`.

But `requestType` and `responseType` are not that. They are Protobuf.js message descriptor objects that describe the structure of `RequestType` and `ResponseType`. @nicolasnoble linked another PR that tried to do this and they declared those values as `Protobuf.Type`, which I think is the actual correct type.